DevoxxClientMobile/src/main/java/com/devoxx/views/cell/ScheduleCell.java

```diff
@@ -320,4 +321,8 @@ private StackPane createIndicator(SessionVisuals.SessionListType style, boolean
             return node;
         }
     }
+
+    private int favoriteThreshold() {
+        return service.getConference().getEventType() == Conference.Type.DEVOXX ? 10 : 3;
```

Comment:
There is no chance that `service.getConference()` returns null, is there?

Reply:
It shouldn't be because this view is only visible after a conference has been selected.